lib/nnue_training_data_formats.h

```diff
@@ -7577,6 +7578,10 @@ namespace binpack
 
                     if (!m_localBuffer.empty())
                     {
+                        // now shuffle the local buffer
+			static thread_local std::mt19937 g(std::random_device{}());
```

Comment:
Usually mt199937_64 is faster and we don't really target 32 bit systems. If the currently version is as fast for you as master I can merge as is.

Reply:
yes, no noticeable slowdown.